privacyidea/static/components/config/controllers/configControllers.js

```diff
@@ -79,6 +80,7 @@ myApp.controller("policyDetailsController", function($scope, $stateParams,
     $scope.scopes = [];
     $scope.viewPolicyTemplates = false;
     $scope.action_filter = "";
+    $scope.inputPattern = resourceNamePattern;
```

Comment:
What about the idea to define the different patterns on the ``rootscope``. This way we could avoid defining a new pattern in each controller and simply use the corresponding pattern in the HTML template?

Reply:
sorry, missed that You already reviewed this PR, i force-pushed a new version which already fixes some of Your remarks.
Also the constants only get injected into the `mainController` and added to the current scope there. I read somewhere that you should not use the root scope but i am just an angluarjs beginner.